constitution.md

```diff
@@ -271,3 +271,7 @@ B-Soc:
 
 There will be no temporary VP-Finance, and that role will fall under the 
 President (since many of its roles should not be active yet).
+
+8.0 Amending the Constitution
```

Comment:
We already have an 8.0. We should probably rename this section 9.0. Also add heading.